### ISSUE TITLE
ng-repeat not working in HTML tbody

### ISSUE THREAD

**Issue by peteford** [angular/angular.js#15798]
***Note*: for support questions, please use one of these channels: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#question. This repository's issues are reserved for feature requests and bug reports.**

**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Nested ng-repeat tags inside <table> elements render no data.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar (template: http://plnkr.co/edit/tpl:yBpEi4).**

I have a list of items, each of which has a name and one or more "category" objects attached; each category has a name and zero or more "comment" objects. I need to render this in a table, so I have this (please see HTML source if this renders as a table):

<table>
  <thead>
    <th>Item</th>
    <th>Category</th>
    <th>Comment</th>
  </thead>
  <tbody>
    <ng-repeat ng-repeat="(itemIx, item) in itemList track by itemIx">
      <ng-repeat ng-repeat="(catIx, category) in item.categoryList track by catIx">
        <tr ng-repeat="comment in category.commentList track by $index">
          <td>{{item.name}}</td>
          <td>{{category.name}}</td>
          <td>{{comment}}</td>
        </tr>
      </ng-repeat>
    </ng-repeat>
  </tbody>
</table>

**What is the expected behavior?**

I expect to see something like this:

Item  Category  Comment
item1 cat1  comment1
item1 cat1 comment2
item1 cat2 comment1
Item2 cat1 comment1
...etc.

However no matter what data is in my list of items, I get an empty table. Inspecting the page element with browser tools shows only a <!-- ng-repeat ... ->comment at the location. I can show the nested ng-repeat outside a table works -- this works exactly the way I expected:

    <ng-repeat ng-repeat="(itemIx, item) in itemList track by itemIx">
      <ng-repeat ng-repeat="(catIx, category) in item.categoryList track by catIx">
        <p ng-repeat="comment in category.commentList track by $index">
          {{item.name}} - {{category.name}} - {{comment}}
        </p>
      ...

**What is the motivation / use case for changing the behavior?**

It is currently impossible to get my data rendered in a table. I need a solution to this for the client.

**Which versions of AngularJS, and which browser / OS are affected by this issue? Did this work in previous versions of AngularJS? Please also test with the latest stable and snapshot (https://code.angularjs.org/snapshot/) versions.**

AngularJS 1.5.9 - Chrome browser/Windows 7. The ability to use <ng-repeat> as a tag was not available in earlier versions (I upgraded to 1.5.9 specifically for this ability). I am unable to update to 1.6.3 at this time (project manager's configuration control rules) but I can request that we update if there is a solution in a newer version.

**Other information (e.g. stacktraces, related issues, suggestions how to fix)**

I'm not familiar enough with the internal working of AngularJS to be able to suggest a cause or a fix.

**Comment by gkalpak** [comment]
Duplicate of #8128, #8664 and numerous others.

**tl;dr**
It's the browser that moves the content outside of the table, tbody, tr, etc. Angular can't do anything about it.
